pom.xml

```diff
@@ -665,6 +667,27 @@
                     </execution>
                 </executions>
             </plugin>
+
+            <plugin>
+                <groupId>org.owasp</groupId>
+                <artifactId>dependency-check-maven</artifactId>
+                <version>3.1.2</version>
+                <configuration>
+                    <failBuildOnAnyVulnerability>${dropwizard.check.fail-owasp}</failBuildOnAnyVulnerability>
+                    <skip>${dropwizard.check.skip-owasp}</skip>
+                    <showSummary>true</showSummary>
+                    <format>ALL</format>
```

Comment:
Do we really need all report formats here? I think HTML (default) should be sufficient.

https://jeremylong.github.io/DependencyCheck/dependency-check-maven/configuration.html